lte/gateway/c/session_manager/SessionState.h

```diff
@@ -732,6 +734,7 @@ class SessionState {
  private:
   std::string imsi_;
   std::string session_id_;
+  int shard_id_;
```

Comment:
Should this be unsigned?

Reply:
I can make it unsigned, wasn't aware of the specifications for declaring variables, but from what I've seen it makes sense to make variables whose values are solely going to positive or zero unisgned.